### PR TITLE
2681 cancel git clone process if user exits import from git workflow

### PR DESCRIPTION
**Pull request by msivasubramaniaan**
Signed-off-by: msivasubramaniaan [msivasub@redhat.com](mailto:msivasub@redhat.com)

### CODE REVIEW

**Comment by mohitsuman** [comment]
This will Fix #2681

File: src/webview/git-import/gitImportLoader.ts [hunk 1]
```diff
@@ -276,18 +292,65 @@ function validateDevFilePath(event: any) {
...
+                        return reject(err);
+                    }
+                    resolve();
+                });
+            }
+        });
+    });
+};
+
+function deleteDirectory(dir) {
```
Comment (dgolovin) [inline src/webview/git-import/gitImportLoader.ts:333]:
```
const fs = require('fs')

// directory path
const dir = 'temp'

// delete directory recursively
fs.rmdir(dir, { recursive: true }, err => {
  if (err) {
    throw err
  }

  console.log(`${dir} is deleted!`)
})
```

**Review by dgolovin — CHANGES_REQUESTED** [review]

**Review by dgolovin — DISMISSED** [review]

**Review by dgolovin — CHANGES_REQUESTED** [review]
See my comment about deleting directory.

File: src/webview/git-import/gitImportLoader.ts (hunk 1 above)
Comment (msivasubramaniaan) [reply src/webview/git-import/gitImportLoader.ts]: Done with recursive call for removing the directory

**Review by dgolovin — CHANGES_REQUESTED** [review]

**Review by dgolovin — APPROVED** [review]

**Pull request merged**